src/main/fc/fc_core.c

```diff
@@ -205,6 +205,13 @@ void mwArm(void)
             return;
         }
         if (!ARMING_FLAG(PREVENT_ARMING)) {
+            #ifdef USE_DSHOT
+            if (mixerConfig()->reverse_motors == false)
+                pwmWriteDshotCommandAll(getMotorCount(), DSHOT_CMD_ROTATE_NORMAL);
```

Comment:
I was using this is just for the temporarily reversing the motors, a little bit of cruft left over from a previous idea.   I'll fix that into a separate non-permanent variable.